app/controllers/mixins/ems_common_angular.rb

```diff
@@ -171,6 +173,7 @@ def ems_form_fields
       if @ems.connection_configurations.metrics.try(:endpoint)
         metrics_hostname = @ems.connection_configurations.metrics.endpoint.hostname
         metrics_port = @ems.connection_configurations.metrics.endpoint.port
+        metrics_database_name = @ems.connection_configurations.metrics.endpoint.database_name
```

Comment:
Have you checked if this does not blow up for endpoints that dont have a `database_name` ?